scipy/optimize/nnls/nnls.f

```diff
@@ -44,13 +44,16 @@
 C             1     THE SOLUTION HAS BEEN COMPUTED SUCCESSFULLY.
 C             2     THE DIMENSIONS OF THE PROBLEM ARE BAD.  
 C                   EITHER M .LE. 0 OR N .LE. 0.
-C             3    ITERATION COUNT EXCEEDED.  MORE THAN 3*N ITERATIONS. 
+C             3    ITERATION COUNT EXCEEDED.  MORE THAN MAXITER ITERATIONS.
+C     MAXITER THE MAXIMUM ALLOWED NUMBER OF ITERATIONS.
+C             IF NEGATIVE, THE LIMIT IS TAKEN AS THE DEFAULT, 3*N.              
 C   
 C     ------------------------------------------------------------------
-      SUBROUTINE NNLS (A,MDA,M,N,B,X,RNORM,W,ZZ,INDEX,MODE) 
+      SUBROUTINE NNLS (A,MDA,M,N,B,X,RNORM,W,ZZ,INDEX,MODE, MAXITER) 
```

Comment:
I'm not sure what the Fortran standard is, and this is a minor comment, but just noticing that here and elsewhere there's no space after comma between subroutine parameters except for the new parameter added.